src/EventStore.Projections.Core/Services/Processing/EventReaderCoreService.cs

```diff
@@ -262,6 +262,13 @@ public void Handle(ReaderSubscriptionMessage.Faulted message)
             if (!_eventReaderSubscriptions.TryGetValue(message.CorrelationId, out projectionId))
                 return; // unsubscribed
 
+            if (message.Reason.Contains("was expected in the stream"))
+            {
+                // Log without fault the projection
+                _logger.Debug(message.Reason);
```

Comment:
Is debug an appropriate level for this log message? If a customer is not expecting the new behaviour it won't necessarily show up in the logs. Suggest minimum level of Trace

Reply:
set to Trace now